test/node/resendable-call-from-client.js

```diff
@@ -109,3 +112,13 @@ function reconnect(port) {
     });
   });
 }
+
+let calledClose = 0;
+function close() {
+  calledClose++;
+  if (calledClose !== 2) {
```

Comment:
why also not a single line?

Reply:
@aqrln asked to split it into two lines see https://github.com/metarhia/jstp/pull/363#discussion_r208611473.

Reply:
@nechaido I believe @tshemsedinov is talking about braces, not about the increment

Reply:
In that case, I just don't like `if` statements without braces.